cluster-scope/README.md

```diff
@@ -34,6 +37,11 @@ The directories and files in this structure fulfill the following purpose:
 - `kustomization.yaml` : Kustomization file including a single resource, which is a sibling file within the folder. Can pull in `components` for additional bootstrapping.
 - `<resource_kind_singular>.yaml` : **Single** resource manifest with a `.spec` that can be deployed to all the clusters.
 
+#### bundles/
+Bundle related resources by including ( or removing) a single line in the relevant overlay.
+  - `<operator>` : The resource that is being bundled.
+  - `kustomization.yaml` : The list of the resources that belong to the operator.
+
```

Comment:
Let's change `<operator>` to `<bundle-name>` since it doesn't have to be an operator.

We should further clarify when/what to bundle. For instance maybe add something like the following as well: 

> Bundled manifests are manifests that are dependent on each other and should be deployed together on the same cluster. You should create a `bundle` that references all these manifests from a single `kustomization.yaml` file following the structure above. This bundle can then be easily imported to the target cluster's by adding it to `overlays/<env>/<cluster_name>/kustomization.yaml`.